### PR TITLE
Move test-only requirements to test section

### PR DESCRIPTION
**Pull request by igortg**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,16 +19,18 @@ requirements:
...
   run:
+    - python
     - fluids >=0.1.68
     - pandas
-    - python
     - scipy
-    - pint
-    - sympy
     - coolprop
-    - matplotlib
```
Comment (volpatto) [inline recipe/meta.yaml:29]: For me, it looks fine. `thermo` has some plotting features, but maybe it should be delegated to the user instead the lib.

**Review by volpatto — APPROVED** [review]
I see no problem in the request changes. @igortg do you have any other modification or it's ready to merge?

File: recipe/meta.yaml (hunk 1 above)
Comment (volpatto) [reply recipe/meta.yaml:29]: @CalebBell what do you think about moving `matplotlib` to the test's dependency?

Comment (igortg) [reply recipe/meta.yaml:29]: All checks passed, so it's ready to merge

**Comment by CalebBell** [comment]
Hi,
I agree that matplotlib should not be installed with thermo by default; as a practical matter, matplotlib has many binary components which may not be available on some architectures/platforms that might still want to use thermo. I have seen this issue before when I used to have matplotlib as a hard dependency of thermo, so I am approving this PR with thanks to both of you!

As a side note, both `pint` and `CoolProp` are optional dependencies, but CoolProp adds a lot to the library, and pint is pure python so there is little harm from requiring it; pint is not even imported by thermo unless the user imports the thermo.units module explicitly.
Sincerely,
Caleb

**Pull request merged**